frontend/src/app/trees/forests/forest-finder/remove-punc.pipe.ts

```diff
@@ -4,13 +4,13 @@ import { Pipe, PipeTransform } from '@angular/core';
   name: 'RemovePunc'
 })
 export class RemovePuncPipe implements PipeTransform {
-
   transform(value: any, toRemove: Array<string>): any {
     let output = value;
-    toRemove.map((punc) => {
+    if (output) {
```

Comment:
is this for breadcrumbs and routes?

Reply:
@lauraGgit this is for routes, but likely dropped while implenting #102.